mautrix_signal/portal.py

```diff
@@ -328,6 +338,14 @@ async def handle_matrix_message(self, sender: 'u.User', message: MessageEventCon
                 except FileNotFoundError:
                     pass
 
+            # Handle disappearing messages
+            if self.expiration_time:
+                disappearing_message = DisappearingMessage(
+                    self.mxid, event_id, self.expiration_time
+                )
+                await disappearing_message.insert()
+                asyncio.create_task(Portal._expire_event(disappearing_message))
```

Comment:
```suggestion
                asyncio.create_task(self._expire_event(disappearing_message))
```

Reply:
I went with `Portal.` since it is a class method.